### DISCUSSION TITLE
Members Sign up directly in the adadmin script

### DISCUSSION THREAD

**Discussion by Infinityhiteck01** [giuliopons/adadmin#63] [General]
I see that there is an option to have members in your own script be logged into the adadmin script, I want my users to sign up in the adadmin script and sign up that way. I do not see this option currently in the script maybe I have this wrong. Pleas help.

**Comment by Infinityhiteck01** [comment]
Disregard I see its in the demo as Sign in and not register I will fix it on my system.

**Reply by giuliopons** [reply]
Yes. AdAdmin normally has its own users system.
If you want you can bind the login of your already existing site with the login system of AdAdmin as described here #49
